app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -105,7 +105,7 @@ class WalkthroughActivityTest {
   }
 
   @Test
-  fun testWalkthroughFragment_nextBtn_configurationChanged_backBtn_worksCorrectly() {
+  fun testWalkthroughActivity_nextBtn_configChange_backBtn_worksCorrectly() {
```

Comment:
either use `btn` or `button` everywhere.

Reply:
same with `inc` and `dec`

Reply:
done